pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_delete_modules.adoc

```diff
@@ -0,0 +1,17 @@
+[id="delete_modules_{context}"]
```

Comment:
For consistency, I suggest using hyphen instead of underscore as the word separator, other than before {context}

Reply:
done